lib/pinchflat_web/controllers/podcasts/podcast_controller.ex

```diff
@@ -6,8 +6,19 @@ defmodule PinchflatWeb.Podcasts.PodcastController do
   alias Pinchflat.Sources.Source
   alias Pinchflat.Media.MediaItem
   alias Pinchflat.Podcasts.RssFeedBuilder
+  alias Pinchflat.Podcasts.OpmlFeedBuilder
   alias Pinchflat.Podcasts.PodcastHelpers
 
+  def opml_feed(conn, %{}) do
```

Comment:
The second arg here probably be an unused variable just to keep with convention I've set in other controller actions that don't use their params:

```suggestion
  def opml_feed(conn, _params) do
```